backends/xnnpack/CMakeLists.txt

```diff
@@ -92,9 +92,8 @@ target_include_directories(
                            ${EXECUTORCH_ROOT}/third-party/flatbuffers/include
 )
 
-set(xnnpack_third_party pthreadpool extension_threadpool cpuinfo)
-
 include(cmake/Dependencies.cmake)
+set(xnnpack_third_party XNNPACK pthreadpool extension_threadpool cpuinfo)
```

Comment:
curious what does this rearrangment do?

Reply:
I just found it weird that we were mutating xnnpack_third_party in cmake/Dependencies.cmake rather than setting it all at once